docs/examples/viz_fine_tuning_gl_context.py

```diff
@@ -0,0 +1,160 @@
+"""
+=======================================================
+Fine-tuning the OpenGL state using shader callbacks
+=======================================================
+
+VTK it’s powerfully, but sometimes we need to get more control about how
+OpenGL will render the actors.  For example, enforcing that deep-test keep
+disabled during the draw call of a specific actor.  This can be useful when
+we need to fine-tuning the performance or create specific visualization
```

Comment:
Remove we need to fine-tuning the performance. Say...

This can be useful when we need to create specialized visualization effects.